omero/annotation_scripts/Import_from_csv.py

```diff
@@ -0,0 +1,858 @@
+            description="Untick this to prevent importing tags specified " +
+                        "in the CSV."),
+
+        scripts.Bool(
+            P_OWN_TAG, optional=True, grouping="2.1", default=False,
+            description="Determines if tags of other users in the group" +
+            " can be used on objects.\n Using only personal tags might " +
+            "lead to multiple tags with the same name in one OMERO-group."),
+
+        scripts.Bool(
+            P_ALLOW_NEWTAG, optional=True, grouping="2.2", default=False,
+            description="Creates new tags and tagsets if the ones" +
+            " specified in the .csv do not exist."),
+
+        scripts.Bool(
+            "Other parameters", optional=True, grouping="3", default=True,
+            description="Ticking or unticking this has no effect"),
+
+        scripts.Bool(
+            P_EXCL_EMPTY, optional=True, grouping="3.1", default=True,
+            description="Exclude a key-value if the value is empty."),
+
+        scripts.String(
+            P_CSVSEP, optional=True, grouping="3.2",
+            description="The separator used in the .csv file. 'guess' will " +
+                        "attempt to detetect automatically which of " +
+                        ",;\\t is used.",
+            values=separators, default="guess"),
+
+        scripts.String(
+            P_SPLIT_CELL, optional=True, grouping="3.3",
+            default="",
+            description="Split cells according to this into multiple " +
+                        "values for a given key."),
+
+        scripts.List(
+            P_EXCL_COL, optional=True, grouping="3.4",
+            default="<ID>,<NAME>,<PARENTS>",
+            description="List of columns in the .csv file to exclude " +
+                        "from the key-value pair import. <ID>" +
+                        " and <NAME> correspond to the two " +
+                        "following parameters. <PARENTS> corresponds " +
+                        "to the six container types.").ofType(rstring("")),
+
+        scripts.String(
+            P_TARG_COLID, optional=True, grouping="3.5",
+            default="OBJECT_ID",
+            description="The column name in the .csv containing the id" +
+                        " of the objects to annotate. " +
+                        "Matches <ID> in exclude parameter."),
+
+        scripts.String(
+            P_TARG_COLNAME, optional=True, grouping="3.6",
+            default="OBJECT_NAME",
+            description="The column name in the .csv containing the name of " +
+                        "the objects to annotate (used if no column " +
+                        "ID is provided or  found in the .csv). Matches " +
+                        "<NAME> in exclude parameter."),
+
+        scripts.Bool(
+            P_ATTACH, optional=True, grouping="3.7", default=False,
+            description="Attach the given CSV to the selected objects" +
+            "when not already attached to it."),
```

Comment:
missing whitespace `objectswhen`

Reply:
Fixed this one and others from all four scripts